src/mds/events/EMetaBlob.h

```diff
@@ -225,26 +226,36 @@ class EMetaBlob {
     bool is_dirty_dft() { return state & STATE_DIRTYDFT; }
     void mark_dirty_dft() { state |= STATE_DIRTYDFT; }
 
-    const list<std::shared_ptr<fullbit> > &get_dfull()   const { return dfull; }
-    const list<remotebit>                  &get_dremote() const { return dremote; }
-    const list<nullbit>                    &get_dnull()   const { return dnull; }
+    const list<fullbit>			&get_dfull() const { return dfull; }
+    list<fullbit>			&_get_dfull() const { return dfull; }
```

Comment:
Why can't it be `list<fullbit>& get_dfull { return dfull; }`? i.e. not const

Reply:
because I want to avoid using non-const dfull in most places.